service/matching/pri_task_writer.go

```diff
@@ -249,3 +254,10 @@ func (w *priTaskWriter) allocTaskIDBlock(prevBlockEnd int64) (taskIDBlock, error
 	}
 	return rangeIDToTaskIDBlock(state.rangeID, w.config.RangeSize), nil
 }
+
+func (w *priTaskWriter) currentTaskIDBlock() taskIDBlock {
+	return taskIDBlock{
+		start: atomic.LoadInt64(&w.lastTaskIDBlock.start),
+		end:   atomic.LoadInt64(&w.lastTaskIDBlock.end),
+	}
```

Comment:
This is un-used (CodeCov tells me), but I think https://github.com/temporalio/temporal/pull/7469 might change that.